app/views/home/index.html.erb

```diff
@@ -14,5 +14,33 @@
       <p>Looks like you don't have any poems yet.</p>
     <% end %>
     <%= link_to t("poems.actions.new"), new_poem_path, class: "btn button" %>
+    <h2>Your Submissions</h2>
```

Comment:
Ideally, yup. Once I made sure I had a working translation file I moved onto more of the functionality in this PR. That's definitely something I'll be more mindful of going forward.